index.rst

```diff
@@ -6,24 +6,474 @@
+
+Remember to set the PREOPS ticket to done.
+
+
+Portal tutorial format and style
+================================
+
+The portal tutorials are written in RST format and are kept within the data release documentation at
+`dp0-2.lsst.io/tutorials-examples <https://dp0-2.lsst.io/tutorials-examples/index.html>`__.
+
+All portal tutorials should have a descriptive title, and list the contact authors, the date last verified to run, and the targeted learning level.
+A brief narrative introduction to the tutorial should be provided at the top of the page.
+
+The rest of the portal tutorial should be divided into sequentially numbered steps and sub-steps.
+Use descriptive text and screenshots to demonstrate what the user should do.
+Augment screenshots with arrows or circles to guide the users attention as needed.
+
+Ensure that any ADQL is put into code boxes in RST so that users may copy-and-paste whenever possible.
+
+It is very common, but not mandatory, to end all portal tutorials with a section called "Exercises for the learner" with suggestions of
+how the user can make changes to the tutorial test options, or take the next step forward on their own.
+
+
+Git branch, merge, and review policy for portal tutorials
+=========================================================
+
+The following applies when creating or updating tutorials in the ``dp0-2_lsst_io`` repository of the ``lsst`` GitHub Org.
+
+Develop new tutorials (or update existing ones) in a new branch, created from ``main``, and named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+Typically, only one tutorial is updated per ticket branch.
+
+Make commits and push changes to the ``dp0-2_lsst_io`` repo until work is complete, then open a pull request to ``main``.
+
+Request a review from at least one Rubin staff member, and ensure that all of their comments and suggestions are incorporated and/or addressed.
+After the reviewer has approved the PR, rebase and merge to the ``main`` branch of ``dp0-2_lsst_io`` and delete the branch.
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+
+Stretch goals
+=============
+
+Work is ongoing in these areas.
+
+Notebook metadata
+-----------------
+
+Embed notebook metadata (e.g., version, skills, packages) in a way that can be scraped and used to auto-generate the `README.md <http://readme.md/>`_ file or a Table of Contents, to emable users to browse notebook contents. 
+
+Narrative voice
+---------------
+
+Adopt a best practice for narrative instructions: should we use terms like "you" or "let's"? Or should a more passive voice be adopted?
+
+Accessibility
+-------------
+
+Improve tutorials' accessibility to people with visual disabilities by finding and implementing, e.g., screen reader compatibility software, data sonification packages.
+
+Translations
+------------
+
+At minimum, translate any undergraduate-level tutorials into Spanish.
```

Comment:
Can we offer tools or some sort of help for people to do this? For example, should Rubin buy a tool (e.g., deepl.com) or pay people to do it?